README.md

```diff
@@ -2,6 +2,46 @@
 
 Welcome to the Woovi Java SDK! This SDK provides convenient access to the Woovi REST API, allowing you to easily integrate Woovi's video upload and management services into your Java applications.
 
+## Docs
+
+How do I use it? You can have a look in the following example:
+
+```java,no
+ApiClient defaultClient = Configuration.getDefaultApiClient();
+defaultClient.setBasePath("https://api.woovi.com");
+
+// Configure API key authorization: AppID
+ApiKeyAuth AppID = (ApiKeyAuth) defaultClient.getAuthentication("AppID");
```

Comment:
why need force type here?

Reply:
`getAuthentication` returns `Authentication` class, and it needs to be `ApiKeyAuth` to set the App ID